CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ All notable changes to [bpmn-js-properties-panel](https://github.com/bpmn-io/bpm
 
 ___Note:__ Yet to be released changes appear here._
 
+`FIX`: trim trailing whitespaces from all input fields except expressions ([bpmn-io/properties-panel#401](https://github.com/bpmn-io/properties-panel/pull/401), [bpmn-io/properties-panel#402](https://github.com/bpmn-io/properties-panel/pull/402))

```

Comment:
Wasn't there an upstream feature request? If so, we should link that one (as a primary reference), I think it was [this one](https://github.com/bpmn-io/properties-panel/issues/309).

Also: I propose we frame this as a feature. It does not fix a bug, but improves the UX of overall property editing.